<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>773</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="bgWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 111, 113);</string>
   </property>
   <widget class="QLabel" name="locationLabel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>25</y>
      <width>600</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>32</pointsize>
      <weight>10</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
background-color: rgb(255, 255, 255);
font: 87 32pt &quot;Arial Black&quot;;
color:rgb(0, 111, 113);</string>
    </property>
    <property name="text">
     <string>Welcome to DeskFood</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="emailEdit">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>250</y>
      <width>300</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;
background-color: rgba(255, 255, 255, 0);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordEdit">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>350</y>
      <width>300</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;
background-color: rgba(255, 255, 255, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>225</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 87 12pt &quot;Arial Black&quot;;
color:rgb(0, 111, 113);</string>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>325</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 87 12pt &quot;Arial Black&quot;;
color:rgb(0, 111, 113);</string>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loginButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>425</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
color: rgb(255, 255, 255);
background-color: rgb(0, 111, 113);
font: 87 12pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QPushButton" name="registerButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>425</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
background-color: rgb(0, 111, 113);
color: rgb(255, 255, 255);
font: 87 12pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>200</y>
      <width>400</width>
      <height>300</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
   </widget>
   <zorder>frame</zorder>
   <zorder>locationLabel</zorder>
   <zorder>emailEdit</zorder>
   <zorder>passwordEdit</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>loginButton</zorder>
   <zorder>registerButton</zorder>
  </widget>
 </widget>
 <tabstops>
  <tabstop>emailEdit</tabstop>
  <tabstop>passwordEdit</tabstop>
  <tabstop>loginButton</tabstop>
  <tabstop>registerButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
